packages/patternfly-4/react-core/src/components/Select/SelectMenu.tsx

```diff
@@ -133,21 +137,21 @@ export class SelectMenu extends React.Component<SelectMenuProps> {
         {({ variant }) => (
           <React.Fragment>
             {variant !== SelectVariant.checkbox && (
-              <ul {...props} className={css(styles.selectMenu, className)} role="listbox">
+              <ul {...props} className={css(styles.selectMenu, className)} role="listbox" {...maxHeight && {style:({ maxHeight, overflow: 'auto' })}}>
```

Comment:
Not a giant fan of using a style tag as our implementation. Can we add a custom React class so the consumer can override it?

Reply:
Ah, I realize now there's not much to override other than the `overflow` type. Let's just move the `{...props}` to the end in case a `style` tag is passed in `props`.

Reply:
I'm not sure how we could assign a class and still have a variable the user could customize.